Arrangement note changes

gwiedeman opened the pull request:
Reworks rules around for the arrangement note per #48. This would be a major change and requires substantial community feedback. This is an early draft and I would expect significant changes throughout the process.

Currently, arrangement note practice is inconsistent and provides little value to users. Often archivists merely list series as if the note was serving as a "Table of Contents" for a collection, which is now often done automatically by access systems. Previous exclusion 3.2.1 was also inconsistently applied. This change encourages archivist to document the source or arrangement and order, but the boundaries between the arrangement note and processing note are now more unclear and need to be better defined.

These changes makes the arrangement note more consistent with the DACS Statement of Principles, including the discussions about identifying aggregations of records, original order and arrangement as archival context, as well as Principles 2, 4, and 6.

Changes:
* Adds new Rule 3.2.5 which explicitly states that an arrangement note is not necessary if an access system can list series.
* Adds screenshot examples with alt text for ArchivesSpace and Arclight as well as a separate directory structure for images.
* Removes Exclusion 3.2.1 which relegated all description of the source of arrangement to an unrequired processing note that is rarely implemented.
* Raises the priority of previous Rule 3.2.4, new Rule 3.2.3 that provides information about component order from "optionally" to "preferably."
* Fixed indentation and removed some redundant examples.
* Added an example where a collection is an arrangement of accessions.

gwiedeman committed:
Update 02_system_of_arrangement.md

Removed the exclusion.

gwiedeman committed:
Update 02_system_of_arrangement.md

Adding new 3.2.3 that states arrangement notes are not needed if an access system provides all the necessary information.

gwiedeman committed:
added screenshots for arragement note

gwiedeman committed:
Update 02_system_of_arrangement.md

Should fix image paths to screenshots.

gwiedeman committed:
Update 02_system_of_arrangement.md

Testing explicit path to make sure this works.

gwiedeman committed:
moved images as they apparently only work in the immediate directory?

gwiedeman committed:
does work loading images from other paths

gwiedeman committed:
Update 02_system_of_arrangement.md

Does this fix indents?

gwiedeman committed:
Update 02_system_of_arrangement.md

indents

gwiedeman committed:
Update 02_system_of_arrangement.md

gwiedeman committed:
Update 02_system_of_arrangement.md

How does github recognize indents?

gwiedeman committed:
Update 02_system_of_arrangement.md

Fixed indentation

gwiedeman committed:
Update 02_system_of_arrangement.md

gwiedeman committed:
Update 02_system_of_arrangement.md

gwiedeman committed:
Update 02_system_of_arrangement.md

gwiedeman committed:
Update 02_system_of_arrangement.md

gwiedeman committed:
Update 02_system_of_arrangement.md

gwiedeman committed:
Update 02_system_of_arrangement.md

gwiedeman committed:
Update 02_system_of_arrangement.md

gwiedeman committed:
Update 02_system_of_arrangement.md

gwiedeman committed:
Update 02_system_of_arrangement.md

gwiedeman committed:
Update 02_system_of_arrangement.md

regineheberlein reviewed (changes requested):
I would suggest removing or replacing the second example with one that's less sensitive

kbogan committed:
Update 02_system_of_arrangement.md

Removing this example per TS-DACS discussion. We want to reevaluate this example as part of the larger example project.
